src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -53,40 +53,66 @@ void ApplyConventions(ReadOnlySettings settings)
                 return;
             }
 
-            // Source: https://github.com/ravendb/ravendb/blob/f56963f23f54b5535eba4f043fb84d5145b11b1d/Raven.Client.Lightweight/Document/DocumentStore.cs#L129
-            var ravenDefaultResourceManagerId = new Guid("e749baa6-6f76-4eef-a069-40a4378954f8");
-
-            if (store.ResourceManagerId != ravenDefaultResourceManagerId)
+            bool suppressDistributedTransactions;
+            if (settings.TryGet("Transactions.SuppressDistributedTransactions", out suppressDistributedTransactions) && suppressDistributedTransactions)
             {
-                Logger.Warn("Overriding user-specified documentStore.ResourceManagerId. It's no longer necessary to set this value while using NServiceBus.RavenDB persistence. Consider using busConfiguration.CustomizeDocumentStore(Action<IDocumentStore customize) if this is absolutely necessary, but it is not recommended.");
+                store.EnlistInDistributedTransactions = false;
             }
-            if (!(store.TransactionRecoveryStorage is VolatileOnlyTransactionRecoveryStorage))
+            else
             {
-                Logger.Warn("Overriding user-specified documentStore.TransactionRecoveryStorage. It's no longer necessary to set this value while using NServiceBus.RavenDB persistence. Consider using busConfiguration.CustomizeDocumentStore(Action<IDocumentStore customize) if this is absolutely necessary, but it is not recommended.");
-            }
-
-            var resourceManagerId = settings.Get<string>("NServiceBus.LocalAddress") + "-" + settings.Get<string>("EndpointVersion");
-            store.ResourceManagerId = DeterministicGuidBuilder(resourceManagerId);
+                // Source: https://github.com/ravendb/ravendb/blob/f56963f23f54b5535eba4f043fb84d5145b11b1d/Raven.Client.Lightweight/Document/DocumentStore.cs#L129
+                var ravenDefaultResourceManagerId = new Guid("e749baa6-6f76-4eef-a069-40a4378954f8");
```

Comment:
I think it's highly unlikely Raven would change that Guid for no good reason. And even given that, the guid is used to:

1. If docStore already initialized (non-preferred scenario anyway) to verify that the DTC settings are acceptable. So basically we're asking "Did you set the RMId to *something else*? In that case they still shouldn't be using the old Raven default value.
2. If we are initializing, we are about to override the RMid, so we use the test to provide a helpful warning if they already set a RMId that they really don't need to. So if Raven changes the value, this check breaks and they don't get an ignorable warning. No biggie.